components/raftstore/src/store/worker/region.rs

```diff
@@ -721,7 +734,9 @@ where
         self.handle_pending_applies();
         self.clean_stale_tick += 1;
         if self.clean_stale_tick >= STALE_PEER_CHECK_TICK {
-            self.ctx.clean_stale_ranges();
+            if !self.ctx.ingest_maybe_stall() {
+                self.ctx.clean_stale_ranges();
```

Comment:
just trying to understand. Is it true that in case of write stall, the worker thread will wait for the next tick to reschedule the stale range cleanup?

Reply:
I'm wondering whether we should notify PD to stop scheduling more region migration, if there is a backlog of clean_stale_range backlog.

Reply:
There are some limit in PD. If we stop consume the request to region-worker, PD will stop scheduling more region migration. Please check that whether there is other negative impact @BusyJay